resources/daemonset.yaml

```diff
@@ -119,7 +114,16 @@ spec:
                 fieldRef:
                   apiVersion: v1
                   fieldPath: spec.nodeName
+            - name: DATA_PATH
+              value: /cosi-secret-dir
+            - name: MAX_VOLUMES
+              value: "100"
```

Comment:
still unclear to me how this is relevant to cosi and buckets

Reply:
DATA_PATH is the path at which directories for each volume are created.

The workflow follows as:
1. The CO sends a request to NodePublishVolume with the path at which we should mount the secret (target_path), this is given by the podspec
2. We use the volume context to get the secret and protocol
3. We create a folder at DATA_PATH/VOLUME_ID, where the volume ID is provided by the CO in the request
4. We mount the "DATA_PATH/VOLUME_ID" folder to the TargetPath

Reply:
Understood. But the limit of 100 is intended to limit the number of buckets _per node_ to 100/2 (2 vols per BAR) -- is this correct?